doc/source/building/index.rst

```diff
@@ -161,7 +161,8 @@ your system.
     This is needed even if you use the MinGW-w64 or Intel compilers, in order
     to ensure you have the Windows Universal C Runtime (the other components of
     Visual Studio are not needed when using Mingw-w64, and can be deselected if
-    desired, to save disk space).
+    desired, to save disk space). The recommended version of the UCRT is
+    10.0.22621.0.
```

Comment:
Recommending such a specific version is probably not helpful. I haven't seen this go wrong before, so I assume any recent enough version >= this one will work. Do you have a reason to believe that that isn't the case? If not, let's use `>=` here.

Reply:
Agree. Will change this to >=.